Codegen/test/Function.cs

```diff
@@ -20,13 +20,13 @@ public class Function {
 		}
 
 		/// <summary>Creates function.</summary>
-		public Function(QualifiedName name, IList<ParameterDescriptionEntry> inputs, ParameterDescriptionEntry returning, AccessModifier visibility = AccessModifier.Private)
+		public Function(QualifiedName name, IList<ParameterDescriptionEntry> inputs, ParameterDescriptionEntry returning, AccessModifier visibility = AccessModifier.Local)
```

Comment:
The goal of this issue was not to change the default visibility of function from Private to Local.
It was supposed to be done in #1081.

So I propose to change the goal of this issue and referenced it in #1081

Reply:
The default visibility is indeed now Local instead of Private but this doesn't affect the way functions or procedures (or types) are treated. All tests are done against Public (if vis == AccessModifier.Public) or against not Public (if vis != AccessModifier.Public). The Local access modifier is not used directly and so is considered same as Private.
Documentation and error messages are impacted but not behavior.